candig/server/network/__init__.py

```diff
@@ -18,8 +18,8 @@ def getInitialPeerList(filePath, logger=None):
 
     This file has one URL per line and can contain newlines and comments.
 
-        # Main ga4gh node
-        http://1kgenomes.ga4gh.org
+        # Main candig node
+        http://1kgenomes.candig.org
```

Comment:
This should also remain ga4gh.

Reply:
Ah right, though I just realized that this http://1kgenomes.ga4gh.org does not exist anymore either..

Reply:
As per our previous conversation, we can probably just get rid of this part altogether.. since none of the links is valid anymore